engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade41900to41910.java

```diff
@@ -61,6 +66,25 @@ public InputStream[] getCleanupScripts() {
             throw new CloudRuntimeException("Unable to find " + scriptFile);
         }
 
-        return new InputStream[] {script};
+        return new InputStream[]{script};
+    }
+
+    private void addIndexes(Connection conn) {
+        DbUpgradeUtils.addIndexIfNeeded(conn, "vm_stats", "vm_id");
+    }
+
+    @Override
+    public void updateSystemVmTemplates(Connection conn) {
+        LOG.debug("Updating System Vm template IDs");
+        initSystemVmTemplateRegistration();
+        try {
+            systemVmTemplateRegistration.updateSystemVmTemplates(conn);
+        } catch (Exception e) {
+            throw new CloudRuntimeException("Failed to find / register SystemVM template(s)");
+        }
+    }
+
+    private void initSystemVmTemplateRegistration() {
+        systemVmTemplateRegistration = new SystemVmTemplateRegistration("");
```

Comment:
Is this relevant to the PR?

Reply:
These are common pattern of changes, without this CloudStack doesn't check/auto-configure/install/upgrade systemvmtemplates.